Corelight Zeek Modeling & Parsing (#23626)
demisto/content: Packs/CorelightZeek/README.md

# Corelight Zeek
This pack includes Cortex XSIAM content. 
## Configuration on Server Side
You need to configure Corelight Sensor to forward Zeek Syslog messages.

> 1. Open Corelight Sensor UI, at the left bar choose "Configuration".

Reviewer: ```suggestion
1. Open the Corelight Sensor UI, and on the left menu bar click **Configuration**.
```

> 2. Open the "Export" tab and scroll down to "Export to Syslog";

Reviewer: ```suggestion
2. Open the **Export** tab and scroll down to **Export to Syslog**.
```

>     * Under "Syslog Server"- Set you XSIAM IP and Port numbers.

Reviewer: ```suggestion
    * Under "Syslog Server"- Set your XSIAM IP and Port numbers.
```

>     * Under "Syslog Format"- Pick the "Default" option.

Reviewer: ```suggestion
    * Under **Syslog Format**- Pick the "Default" option.
```

>     * Under "Syslog Facility"- Set your Syslog Facility, E.g. Local0.

Reviewer: ```suggestion
    * Under "Syslog Facility"- Set your Syslog Facility, e.g., Local0.
```

>     * Under "Syslog Severity"- Select the severity for the logs you will sent, E.g. Info.

Reviewer: ```suggestion
    * Under "Syslog Severity"- Select the severity for the logs you will sent, e.g., Info.
```

> 3. Open the "Maintain" tab.

Reviewer: ```suggestion
3. Open the **Maintain** tab.
```

> 4. Under "Performance Monitoring", enable the "Reporting to Syslog (UDP-Only)" option and set a time interval (E.g. 1min).

Reviewer: ```suggestion
4. Under **Performance Monitoring**, enable the **Reporting to Syslog (UDP-Only)** option and set a time interval (e.g., 1min).
```

> 5. Under "Performance Monitoring", enable the "Reporting Through Zeek Log Streams" option and set a time interval (E.g. 1min).

Reviewer: ```suggestion
5. Under **Performance Monitoring**, enable the **Reporting Through Zeek Log Streams** option and set a time interval (e.g., 1min).
```
